docs/server/README.md

```diff
@@ -1,5 +1,16 @@
 # Discodo Server Documentation
 
+## Urls
+
+Restful
+```
+ http://localhost
```

Comment:
It would be better to use the format ``<host>:<port>`` than localhost.

Reply:
Good